### ISSUE TITLE
701486 Smoke detector and Siren

### ISSUE THREAD

**Issue by Marlark** [mruiter/com.popp#83]
Would it be possible to add the new 701486 to the app?

// Ted

**Comment by JakobWilhelmsson** [comment]
Are you sure that the 701486 is the new smoke detector?
According to their website, it is no longer in production:

https://www.popp.eu/products/sensors/10-year-smoke-sensor/

The smoke detector 700342 seems to be newer, but I can’t see that it is supported in the app either.

**Comment by Marlark** [comment]
Hmm damn didnt see that it was not made anymore. Ordered 3 of them. Knew i should have gotten the other one.

But would have been nice to get it in there anyway 😉

// Ted

Skickades från E-post<https://go.microsoft.com/fwlink/?LinkId=550986> för Windows 10

Från: JakobWilhelmsson<mailto:notifications@github.com>
Skickat: den 1 december 2020 20:14
Till: mruiter/com.popp<mailto:com.popp@noreply.github.com>
Kopia: Marlark<mailto:teddyhq@hotmail.com>; Author<mailto:author@noreply.github.com>
Ämne: Re: [mruiter/com.popp] 701486 Smoke detector and Siren (#83)


Are you sure that the 701486 is the new smoke detector?
According to their website, it is no longer in production:

https://www.popp.eu/products/sensors/10-year-smoke-sensor/<https://eur02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.popp.eu%2Fproducts%2Fsensors%2F10-year-smoke-sensor%2F&data=04%7C01%7C%7Cdfd2a251a01b45bce37d08d8962d588c%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637424468785117029%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=BcttYsrU5B2rY3D4%2FuKbKp%2FyBuNfZtR5BE75dGW1TU0%3D&reserved=0>

The smoke detector 700342 seems to be newer, but I can’t see that it is supported in the app either.

**Comment by Marlark** [comment]
It turns out it is a newer version of the older one. They just hadnt updated their webpage 

https://www.popp.eu/products/smoke-sensor-siren/

**Comment by mruiter** [comment]
For the device to be added I need the device manu and product Ids 

Verstuurd vanaf mijn iPhone

**Comment by Marlark** [comment]
I never tried it but after i watched the webpage i thought wonder if it is possible to join it in anyway. I managed to actually join it in as the smoke detector and siren unit in the app. But there is a couple of minor things that aint working. Looks like they reused some of the id's

Lets se if i can figure this out.

Unitid 64
Manufacturerid 340
Producttypeid 256
Productid 513
Firmwarever 2
Firmwareminorver 1
Hardwarever 1

The things i see right away that aint woring is the batteri status and the button on the firstpage menu. The basic ON and OFF seam to work sins it turn on and off the alarm.

// Ted


Skickat från min Galaxy



-------- Originalmeddelande --------
Från: Marco Ruiter <notifications@github.com>
Datum: 2021-02-19 04:19 (GMT+01:00)
Till: "mruiter/com.popp" <com.popp@noreply.github.com>
Kopia: Marlark <teddyhq@hotmail.com>, Author <author@noreply.github.com>
Ämne: Re: [mruiter/com.popp] 701486 Smoke detector and Siren (#83)


For the device to be added I need the device manu and product Ids

Verstuurd vanaf mijn iPhone

**Comment by Albert1085** [comment]
Hey guys, hey Martin,

because I'm still satisfied with my actual about 4 years old smoke detectors with built-in alarm sirene (article nr. 4001), I just looked up Popp's homepage to see, if they are still offered. But they weren't anymore longer.
Alternatively I saw the above mentioned smoke detector as an successor. Therefore, I'm very interested if it's in the meantime possible to add it to Homey and get it connected via Z-Wave with the existing detectors. 
It would be delighting to get it done! 
Thanks for some more infos about it.
Greets
Albert

**Comment by Albert1085** [comment]
> Hey guys, hey Martin,
> 
> because I'm still satisfied with my actual about 4 years old smoke detectors with built-in alarm sirene (article nr. 4001), I just looked up Popp's homepage to see, if they are still offered. But they weren't anymore longer.
> Alternatively I saw the above mentioned smoke detector as an successor. Therefore, I'm very interested if it's in the meantime possible to add it to Homey and get it connected via Z-Wave with the existing detectors.
> It would be delighting to get it done!
> Thanks for some more infos about it.
> Greets
> Albert

What about adding the new smoke detector of Popp now? Is it possible in the meantime?
Thanks for a reply...
Take care and have a happy easter!
Greets Albert

**Comment by Marlark** [comment]
Yes it works.. Looks like it is just the batteri status that is not working. Looking into a good solution to use the external power addon to my units.

**Comment by Albert1085** [comment]
In case my first answer didn't reach you in fact of the wrong address
***@***.***).....

Thanks so much for your quick response, even just before easter. That's
good to know anyway, that the new smoke detector (Popp article-nr.
701486) already works as an successor of the older one.
The only thing I'm questioning myself about is, which the corresponding
device in the list will it be?

Could you please figure it out for me? Thanks for your advices and support.

Am 03.04.2021 um 12:47 schrieb Marlark:
